package.json

```diff
@@ -76,6 +76,8 @@
     "babel-jest": "^24.5.0",
     "babel-loader": "^8.2.4",
     "commander": "^7.2.0",
+    "axios": "^0.26.1",
+    "dotenv": "^16.0.0",
```

Comment:
I think we may want to create a new package.json file in `component-generator` directory to house its specific dependencies rather than adding them to the global Paragon (library) package.json 🤔

Reply:
Initialized `package.json` in the `component-generator` directory. Updated command to launch the script inside root directory. Moved dependencies to `component-generator`